plugins/h3/h3.cpp

```diff
@@ -66,269 +66,297 @@ static void toSetOf(bool &__isAllResult, size32_t &__lenResult, void *&__result,
 namespace h3
 {
 
-//--------------------------------------------------------------------------------
-//                           ECL SERVICE ENTRYPOINTS
-//--------------------------------------------------------------------------------
+    //--------------------------------------------------------------------------------
```

Comment:
The contents of namespaces aren't generally indented.

Reply:
Whoops - auto format I am afraid - I will fix.